### PR TITLE
#402 html rendering for firefox

### PR DESCRIPTION
**Pull request by krzyk**
#402 
* added html rendering for Firefox
* covered with agenda page test

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1024](https://github.com/zerocracy/farm/issues/1024) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1024](https://github.com/zerocracy/farm/issues/1024) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1371344903); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

**Comment by krzyk** [comment]
@carlosmiranda Hi, I'm going on vacation in 3 days, so I would like to close my tasks before that. You have 3 PRs from me assigned right now: #1024, #1025 , #1027. Could you look at them?

File: src/main/java/com/zerocracy/tk/RsPage.java [hunk 1]
```diff
@@ -165,16 +176,26 @@ private static Response make(final Farm farm, final String xsl,
                 new XeMillis(true)
             )
         );
+        final RsPrettyXml xml =
```
Comment (carlosmiranda) [inline src/main/java/com/zerocracy/tk/RsPage.java]: @krzyk we only use this once, can it remain inlined?

File: src/test/java/com/zerocracy/tk/profile/TkAgendaTest.java [hunk 2]
```diff
@@ -37,20 +38,56 @@
  * @since 0.13
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @todo #402:30min Write tests confirming that awards and profile pages render
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/tk/profile/TkAgendaTest.java]: @krzyk Let's split this puzzle into the three portions described here:

* Tests for Awards
* Tests for Profile
* Tests for RsPage

**Comment by krzyk** [comment]
@carlosmiranda updated

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@yegor256 ping

**Comment by krzyk** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by krzyk** [comment]
@yegor256 please try to merge again (I already reported a bug for this unstable test  some time ago in https://github.com/zerocracy/farm/issues/1032)

**Comment by krzyk** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1024](https://github.com/zerocracy/farm/issues/1024) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thank you

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by ypshenychka** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1024#issuecomment-394138996))

@ypshenychka This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1024](https://github.com/zerocracy/farm/issues/1024) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231024%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**